CMakeLists.txt

```diff
@@ -263,6 +264,14 @@ if(TRITON_BUILD_PYTHON_MODULE AND NOT WIN32)
   target_link_libraries(triton ${CUTLASS_LIBRARIES} ${PYTHON_LDFLAGS})
 endif()
 
+if (NOT "X${TRITON_CODEGEN_BACKENDS}" STREQUAL "X")
+  set(PYTHON_THIRD_PARTY_PATH ${CMAKE_CURRENT_SOURCE_DIR}/python/triton/third_party)
+  foreach(CODEGEN_BACKEND ${TRITON_CODEGEN_BACKENDS})
+    string(TOLOWER ${CODEGEN_BACKEND} lower_codegen_backend)
```

Comment:
why do we need to lower here... why not just make `TRITON_CODEGEN_BACKENDS` lower-case in the first place?

Reply:
Thanks. Removed it.